noxfile.py

```diff
@@ -230,7 +230,27 @@ def lint(session):
     session.run(*isort, silent=SILENT)
 
     session.run("mypy", ".", "--strict", silent=SILENT)
-    session.run("flake8", "--config", ".flake8")
+
+    flake8 = ["flake8", "--config", ".flake8"]
+    if get_current_os() == "Windows":
+        # flake_copyright doesn't support unicode
+        # savoirfairelinux/flake8-copyright#15
+        flake8_skiplist = [
+            "tests/test_apps/config_source_test/dir/config_with_unicode.yaml",
+            (
+                "examples/plugins/example_configsource_plugin/hydra_plugins/"
+                + "example_configsource_plugin/example_configsource_plugin.py"
+            ),
+            "tests/test_apps/config_source_test/structured.py",
+            "hydra/test_utils/config_source_common_tests.py",
+            (
+                "tests/standalone_apps/namespace_pkg_config_source_test/some_namespace/"
+                + "namespace_test/dir/config_with_unicode.yaml"
+            ),
+        ]
+        flake8.extend([f"--extend-exclude={skip}" for skip in flake8_skiplist])
+
+    session.run(*flake8, silent=SILENT)
```

Comment:
1. Can you exclude the file from flake by putting some comment in the file itself?
2. Can you exclude the file from flake by adding the file to an exclude list in the flake config file?